app/models/assignment.rb

```diff
@@ -9,7 +9,7 @@ class Assignment < ActiveRecord::Base
 
   has_many :rubric_criteria,
            -> { order(:position) },
-           class_name: 'RubricCriterion', 
+           class_name: 'RubricCriterion',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.